### PR TITLE
[Docker][CI] Add NNEF dependency to CI images

### PR DESCRIPTION
**Pull request by agoston-mc**
As per https://github.com/apache/tvm-rfcs/pull/108 and https://github.com/apache/tvm/pull/17053 this PR is to update the Docker images to include the NNEF and NNEF-Tools packages.

### CODE REVIEW

**Comment by mshr-h** [comment]
The ci_riscv build error is caused by the recent commit in risc-v-pk repo. I'm working to fix it by pinning the commit id on #17425.

https://ci.tlcpack.ai/blue/organizations/jenkins/tvm-docker/detail/PR-17433/1/pipeline
> [2024-10-01T15:02:50.101Z] 79.34 ../pk/pk.c: Assembler messages:
> [2024-10-01T15:02:50.101Z] 79.34 ../pk/pk.c:122: Error: unknown CSR `ssp'
> [2024-10-01T15:02:50.101Z] 79.34 make: *** [Makefile:336: pk.o] Error 1
> [2024-10-01T15:02:50.101Z] 79.34 make: *** Waiting for unfinished jobs....

File: docker/install/ubuntu_install_nnef.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+set -u
+set -o pipefail
+
+pip3 install \
+  nnef_tools \
```
Comment (mshr-h) [inline docker/install/ubuntu_install_nnef.sh:24]: I think it's better to pin the version.

**Comment by mshr-h** [comment]
Overall looks good to me! Just a minor change is needed.

File: docker/install/ubuntu_install_nnef.sh (hunk 1 above)
Comment (agoston-mc) [reply docker/install/ubuntu_install_nnef.sh]: Done in https://github.com/apache/tvm/pull/17433/commits/a79e15c56a7c18e3270b7bacb700a428a26f5aee

**Review by mshr-h — APPROVED** [review]

**Comment by tqchen** [comment]
Sorry about that, do mind you send a new PR to reintroduce those

**Pull request merged**